### PR TITLE
Set PHP 7.2  as minimum requirement

### LINKED ISSUE #383: Set PHP 7.2 as minimum
Any other version of PHP is no longer supported.
Having this as a minimum requirement is no BC break. Users relying on lower PHP versions will still get the 3.4 release

### PR DESCRIPTION
**Pull request by gisostallenberg**
Also update the requirements for other packages (which does not change the actual packages used).
Fixes #378.
Closes #383.

### CODE REVIEW

**Comment by reyostallenberg** [comment]
This should target version 3.5.0

File: composer.json [hunk 1]
```diff
@@ -16,18 +16,18 @@
...
         "kasperg/phing-github": "0.2.*",
-        "phing/phing": "~2.7",
+        "phing/phing": "^2.7",
         "php-vcr/php-vcr": "1.2.6 - 1.2.7|^1.3.1",
-        "php-vcr/phpunit-testlistener-vcr": "~1.1.2",
-        "phpunit/phpunit": "~4.4",
-        "psr/log": "~1.0",
-        "symfony/yaml": "~2.1"
+        "php-vcr/phpunit-testlistener-vcr": "^1.1.2",
+        "phpunit/phpunit": "^4.8.36",
```
Comment (derrabus) [inline composer.json]: That version of PHPUnit is officially not compatible with php 7.2.

Comment (gisostallenberg) [reply composer.json]: I know, see #387. I'm currently just trying to fix this Travis CI check.

Comment (gisostallenberg) [reply composer.json]: I had to upgrade to at least PHPUnit 6

**Pull request merged**